src/clj/rems/db/resource.clj

```diff
@@ -1,9 +1,7 @@
 (ns rems.db.resource
   (:require [rems.db.core :as db]
-            [rems.InvalidRequestException]
             [rems.util :refer [getx-user-id]])
-  (:import (org.postgresql.util PSQLException)
-           (rems InvalidRequestException)))
+  (:import (org.postgresql.util PSQLException)))
```

Comment:
I prefer using `[]` inside the import like in requires.

Reply:
It's not worth the effort to change. The IDE adds imports automatically and the established convention in this project is to use `()` - there are 23 instances of `(:import (` and only 3 instances of `(:import [`.

Reply:
Then I will do the change to all of them myself. They are obviously someone else's results :)